test/uk/gov/hmrc/transitmovements/controllers/MovementsControllerSpec.scala

```diff
@@ -862,14 +863,15 @@ class MovementsControllerSpec
             contentAsJson(result) mustBe Json.toJson(Vector(response))
         }
 
-        "must return NOT_FOUND if no ids were found" in forAll(Gen.option(arbitrary[OffsetDateTime]), Gen.option(arbitrary[EORINumber])) {
+        "must return empty list if no ids were found" in forAll(Gen.option(arbitrary[OffsetDateTime]), Gen.option(arbitrary[EORINumber])) {
           (updatedSince, movementEORI) =>
             when(mockRepository.getMovements(EORINumber(any()), eqTo(movementType), eqTo(updatedSince), eqTo(movementEORI), eqTo(None)))
               .thenReturn(EitherT.rightT(Vector.empty[MovementWithoutMessages]))
 
             val result = controller.getMovementsForEori(eoriNumber, movementType, updatedSince, movementEORI)(request)
 
-            status(result) mustBe NOT_FOUND
+            status(result) mustBe OK
+            contentAsJson(result) mustBe Json.toJson(Vector.empty[MovementWithoutMessages])
         }
```

Comment:
Can we make sure we have a test that test that we get `NOT_FOUND` if a **movement** is not found (rather than a valid movement with no messages?)

Can we check that the Mongo repository layer does this? Is there a test we can write for this, or do they already exist?